app/src/ui/abort-merge/abort-merge-warning.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react'
+import { Button } from '../lib/button'
+import { ButtonGroup } from '../lib/button-group'
+import { Dialog, DialogContent, DialogFooter } from '../dialog'
+import { Dispatcher } from '../../lib/dispatcher'
+import { PopupType } from '../../lib/app-state'
+import { Repository } from '../../models/repository'
+import { abortMerge } from '../../lib/git'
+import { Octicon, OcticonSymbol } from '../octicons'
+
+interface IAbortMergeWarningProps {
+  readonly dispatcher: Dispatcher
+  readonly repository: Repository
+  readonly onDismissed: () => void
+  readonly currentBranchName: string
+  readonly comparisonBranchName: string
+}
+
+const titleString = 'Confirm abort merge'
```

Comment:
:art: to help these stand out as label resources, maybe we could settle on a suffix that's more specialized than `String`. Something like `titleLabel` or `titleText`?